packages/mdc-fab/_mixins.scss

```diff
@@ -78,14 +80,28 @@
 }
 
 @mixin mdc-fab-ripple($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
   @include mdc-ripple-common($query);
 
   // postcss-bem-linter: define fab
 
   .mdc-fab {
-    @include mdc-ripple-surface($query: $query);
-    @include mdc-ripple-radius-bounded($query: $query);
-    @include mdc-states(on-secondary, $query: $query);
+    @include mdc-ripple-surface(
+      $query: $query, $ripple-target: $mdc-fab-ripple-target);
+    @include mdc-ripple-radius-bounded(
+      $query: $query, $ripple-target: $mdc-fab-ripple-target);
+    @include mdc-states(
+      on-secondary, $query: $query, $ripple-target: $mdc-fab-ripple-target);
+
+    #{$mdc-fab-ripple-target} {
+      @include mdc-feature-targets($feat-structure) {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        overflow: hidden;
```

Comment:
You can use the `mdc-ripple-target-common` mixin I added recently in https://github.com/material-components/material-components-web/pull/4988/files

(The top/left properties are likely necessary for IE 11.)